searchcorespi/src/vespa/searchcorespi/index/ithreadingservice.h

```diff
@@ -53,6 +53,9 @@ namespace searchcorespi::index {
  * tasks to the index field writer executor, so draining logic needs
  * to sync index field inverter executor before syncing index field
  * writer executor.
+ *
+ * TODO: * indexFieldWriter and indexFieldWriter can be collapsed to one. Both need sequencing,
```

Comment:
Did you mean _indexFieldInverter and indexFieldWriter_ ?

Reply:
yes :)